data/campaigns/tutorial/scenarios/02_Tutorial_part_2.cfg

```diff
@@ -382,16 +382,31 @@ A full list of abilities and weapons specials, along with traits, may be found i
             [/show_if]
         [/message]
 
-        [message]
-            speaker=Galdrad
-            message= _ "While none of your recruited units can move yet, you still can. You need more income; there are some villages near the keep you can capture."
-        [/message]
+        [if]
+            [not]
+                [have_location]
+                    terrain=^V
+                [/have_location]
+            [/not]
+            [then]
+                [message]
+                    speaker=Galdrad
+                    message= _ "While none of your recruited units can move yet, you still can. You need more income; there are some villages near the keep you can capture."
+                [/message]
+
+                {PRINT ( _ "Capture a village")}
 
-        {PRINT ( _ "Capture a village")}
+                [disallow_end_turn]
+                    reason=_"You still have to capture a village!"
+                [/disallow_end_turn]
+            [/then]
+        [/if]
     [/event]
 
     [event]
         name=capture
+        first_time_only=no
```

Comment:
This is still active on later turns. If some of your troops die then it can trigger next time you capture a village.

Reply:
Yeah, I have solved it now.